robbot.py

```diff
@@ -12,8 +12,8 @@
 CFG_FILENAME = 'config.json'
 
 load_dotenv()
-TOKEN = os.getenv('DISCORD_TOKEN')
-#TOKEN = os.getenv('LOCAL_TOKEN')
+#TOKEN = os.getenv('DISCORD_TOKEN')
+TOKEN = os.getenv('LOCAL_TOKEN')
```

Comment:
Make sure token is correct, looks like you swapped it here for the test Robbot

Reply:
@DonChalupa  https://github.com/DonChalupa/Robbot/pull/16/commits/6980b9adef711915524cca7a1bff50ed7eea2639